plutus-core/untyped-plutus-core/test/Transform/Inline/Spec.hs

```diff
@@ -0,0 +1,161 @@
+{-# LANGUAGE BlockArguments      #-}
+{-# LANGUAGE NumericUnderscores  #-}
+{-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Transform.Inline.Spec where
```

Comment:
The test module layout is becoming confusing, because we have a `Transform.Inline.Spec`, but then there are also a bunch of inliner tests in `Transform.Simplify` (and the module name doesn't have `Spec`).

Is there a quick way to make things less confusing? If not, open a new issue and add a TODO.

Reply:
https://github.com/IntersectMBO/plutus/issues/7041